doc/src/environment-variables.md

```diff
@@ -28,7 +28,7 @@
   determines the directory that traces will be written too. Traces are of the
   form PID.trace. Traces can be read by the Catapult project [tracing viewer].
 
-- `RUSTUP_UNPACK_RAM` *unstable* (default 400M, min 100M) Caps the amount of
+- `RUSTUP_UNPACK_RAM` *unstable* (default auto or 500M if on a platform such as FreeBSD or NetBSD, min 210M) Caps the amount of
```

Comment:
I specified `210M` since `220000000/1024^2=209.80835` and the previous documentation seemed to refer to `M` as a unit of `mebibytes` instead of `megabytes`, but perhaps it'd be better to refer to the more human-friendly `megabytes` instead.

Reply:
I think I'll change this to explicitly `MB` instead of using the ambiguous `M` to refer to `MiB`